docs/install/create-project-cookieplone.md

```diff
@@ -132,7 +131,7 @@ See the cookiecutter's README for how to [Use options to avoid prompts](https://
 ```
 
 ```{important}
-For {guilabel}`Project Slug`, you must not use any of the Plone core package names listed in [`constraints.txt`](https://dist.plone.org/release/6.0-latest/constraints.txt).
+For {guilabel}`Project Slug`, you must not use any of the Plone core package names listed in [`constraints.txt`](https://dist.plone.org/release/6.1-latest/constraints.txt).
```

Comment:
@mauritsvanrees @davisagli which is preferred here, `6.1-latest` or `6.1.0`? I think `6.1-latest` is preferred.